prisma/schema.prisma

```diff
@@ -11,64 +11,90 @@ datasource db {
 }
 
 model HealthCheck {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
-
+  id         Int      @id @default(autoincrement())
   statusCode Int
   resMsg     String
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @default(now())
```

Comment:
Why is this updatedAt using @default(now()) vs the @updatedAt the other models use?

Reply:
thanks. this should be @updatedAt, it's not actually in use now, and i've completely overlooked it. I will change that to @updatedAt